code/modules/uplink/uplink_items.dm

```diff
@@ -1271,6 +1271,16 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 1
 	surplus = 30
 
+/datum/uplink_item/stealthy_tools/armorpolish
+	name = "Armor Polish"
+	desc = "This two use polish will strengthen clothing to be as strong as a standard-issue armor vest.  \
+			Reinforced with nanite technology, you are able to stay looking good while bashing heads in. \
+			Beware, you can only polish things that go over your jumpsuit or a hat!"
```

Comment:
Optional since I don't have any better word to suggest here but if you can think of something to replace "that go over your jumpsuit", that'd be great.

Reply:
should "over your jumpsuit" be just "in your suit slot"? I'm unaware of how else to word it.

Reply:
Then leave it as is I suppose